openvdb_points/tools/AttributeSet.h

```diff
@@ -305,6 +337,7 @@ class AttributeSet::Descriptor
     size_t insert(const std::string& name, const NamePair& typeName);
     NameToPosMap                mNameMap;
     std::vector<NamePair>       mTypes;
+    NameToPosMap                mGroupMap;
```

Comment:
Added some extra doxygen comments to reflect the above, hopefully that's a little clearer now.